gravatar-quickeditor/src/test/java/com/gravatar/quickeditor/ui/oauth/OAuthViewModelTest.kt

```diff
@@ -0,0 +1,97 @@
+package com.gravatar.quickeditor.ui.oauth
+
+import app.cash.turbine.test
+import com.gravatar.quickeditor.data.service.WordPressOAuthService
+import com.gravatar.services.ErrorType
+import com.gravatar.services.Result
+import io.mockk.coEvery
+import io.mockk.mockk
+import junit.framework.TestCase.assertEquals
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.TestDispatcher
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.junit.Before
+import org.junit.Test
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class OAuthViewModelTest {
+    private val wordPressOAuthService = mockk<WordPressOAuthService>()
+
+    private lateinit var dispatcher: TestDispatcher
+    private lateinit var viewModel: OAuthViewModel
+
+    @Before
+    fun setup() {
+        dispatcher = StandardTestDispatcher()
+        Dispatchers.setMain(dispatcher)
```

Comment:
💡Should we create a rule like the suggested [here](https://developer.android.com/kotlin/coroutines/test#setting-main-dispatcher)? Or at least invoke [resetMain](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-test/kotlinx.coroutines.test/reset-main.html) in a `@After` method?

Reply:
Good call, I will add a rule for it!